src/bandersnatch_storage_plugins/s3.py

```diff
@@ -331,6 +331,15 @@ def mkdir(
             path = self.PATH_BACKEND(path)
         path.joinpath(self.PATH_BACKEND.keep_file).touch()
 
+    def scandir(self, path: PATH_TYPES) -> Generator[S3DirEntry, None, None]:
+        """Read entries from the provided directory"""
+        if not isinstance(path, self.PATH_BACKEND):
+            path = self.PATH_BACKEND(path)
+        for p in path._accessor.scandir(path):
+            if p.name == ".s3keep":
+                continue
```

Comment:
Is this special to s3 - Can we link to why we skip this for less s3 versed people? Or just a comment.

Reply:
Yes, but for swift there's `.swiftkeep`. Actually, I should have used `self.PATH_BACKEND.keep_file` here to align with the `mkdir()`, which have explained just lines above.

Reply:
Just fixed this. ~~Oh no the actions got started again...~~